ndless-sdk/tools/zehn_loader/Makefile

```diff
@@ -1,18 +1,28 @@
 GXX := "$(shell (which arm-elf-g++ arm-none-eabi-g++ arm-linux-gnueabi-g++ | head -1) 2>/dev/null)"
 OBJCOPY := "$(shell (which arm-elf-objcopy arm-none-eabi-objcopy arm-linux-gnueabi-objcopy | head -1) 2>/dev/null)"
+CHOST := $(shell (echo $(shell basename $(GXX)) | sed 's/-g++$$//'))
```

Comment:
What is CHOST used for here? Normally CHOST is the prefix of the host toolchain, which is most likely not arm.